http/bucket_service.go

```diff
@@ -148,6 +149,18 @@ type retentionRule struct {
 	EverySeconds int64  `json:"everySeconds"`
 }
 
+func (rr *retentionRule) RetentionPeriod() (time.Duration, error) {
+	t := time.Duration(rr.EverySeconds) * time.Second
+	if t < time.Second {
```

Comment:
The current minimum retention period for a bucket is one hour. This should probably also be a constant somewhere.